ballerina-shell/modules/shell-cli/src/main/java/io/ballerina/shell/cli/handlers/HelpCommand.java

```diff
@@ -47,13 +57,25 @@ public void run(String... args) {
             String helpFile = PropertiesLoader.getProperty(HELP_FILE);
             String helpContent = FileUtils.readResource(helpFile);
             ballerinaShell.outputInfo(helpContent);
+
         } else {
             try {
-                StringBuilder content = new StringBuilder();
-                helpProvider.getTopic(args, content);
-                ballerinaShell.outputInfo(content.toString());
+                bbeHelpProvider.readBbeFile();
+            } catch (HelpProviderException e) {
+                ballerinaShell.outputError("Error Occurred While Executing The Command");
```

Comment:
Let's use proper sentence case.

Reply:
Updated in b369170151717249ec60d86bfbeab195a30174d8